documentation/modules/configuring/proc-migrating-clusters-node-pools.adoc

```diff
@@ -47,6 +47,8 @@ spec:
         size: 100Gi
         deleteClaim: false
 ----
++
+WARNING: To migrate a cluster while preserving the names of its nodes and resources, the node pool name must be `kafka` and the `strimzi.io/cluster` label must use the name of the `Kafka` resource. Otherwise, nodes and resources are created with new names, including the persistent volume storage used by the nodes. Consequently, your previous data may not be available as expected.        
```

Comment:
Would it be worth making it more clear that the data would be lost as well?

```suggestion
WARNING: To migrate a cluster while preserving the names of its nodes, resources, and its data, the node pool name must be `kafka` and the `strimzi.io/cluster` label must use the name of the `Kafka` resource. Otherwise, nodes and resources are created with new names, including the persistent volume storage used by the nodes. Consequently, your previous data may not be available as expected.        
```